streams/src/test/java/org/apache/kafka/streams/integration/KStreamAggregationDedupIntegrationTest.java

```diff
@@ -126,9 +126,9 @@ public void whenShuttingDown() throws IOException {
     @Test
     public void shouldReduce() throws Exception {
         produceMessages(System.currentTimeMillis());
-        groupedStream
-            .reduce(reducer, "reduce-by-key")
-            .to(Serdes.String(), Serdes.String(), outputTopic);
+        groupedStream.reduce(reducer, "reduce-by-key")
+                .toStream()
+                .to(Serdes.String(), Serdes.String(), outputTopic);
```

Comment:
Shouldn't this be `to(outputTopic, Produced.with(...))` ?

Reply:
I plan to remove the part related to KIP-182 in another PR. This is just for KTable's deprecated ones.